certified-connectors/App Power Forms/apiDefinition.swagger.json

```diff
@@ -0,0 +1,647 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "App Power Forms",
+    "description": "Use this connector from App Power LLC to easily include external stakeholders in your business processes. The connector makes it very easy to create a form within the designer. You can start with a free trial license and acquire a production license at https://apppower.net.",
+    "version": "1.0",
+    "contact": {
+      "name": "App Power",
+      "url": "https://apppower.net",
+      "email": "support@apppower.net"
+    }    
+  },
+  "host": "apppower-forms-dev.azurewebsites.net",
```

Comment:
@TerhoAntila We do not certify the connectors having dev/staging/test host URL. Please provide a production version of host to move forward for certification.

Reply:
How did that end up there 😅 Will fix...